### DISCUSSION TITLE
How to set the OS window title in `kitty.conf`?

### DISCUSSION THREAD

**Discussion by eval-exec** [kovidgoyal/kitty#6564] [Q&A] [answered]
I want to set specific window title for kitty, and I only can do that by `kitty --title balabala`.

But I want to specify a custom title in `kitty.conf`, then I will got my specific title after `kitty`.

**Comment by kovidgoyal** [comment, marked as answer]
You cannot. kitty.conf is common to all instances of kitty. So you
cannot set instance specific things there.
